### PR TITLE
Add P2 icon

### PR DESCRIPTION
**Pull request by mindgraffiti**
Parent issue: https://github.com/wordpress-mobile/WordPress-iOS/issues/15264

This PR updates the .proj files so it's compatible with Xcode ~12~ 11, applies the Xcode recommended project settings updates, adds the P2 icon, and updates the podspec file.

<a href="https://user-images.githubusercontent.com/1062444/100914100-37c0fd00-3498-11eb-8d4c-b7ac2545f509.png"></a>

### To test
1. Check out this branch
2. `rake dependencies`
3. `rake gen`
4. Build and run the Demo
5. See the new P2 icon listed in the demo

### CODE REVIEW

File: Gridicons.podspec [hunk 1]
```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |s|
   s.name         = "Gridicons"
-  s.version      = "1.0.2"
+  s.version      = "1.0.3-beta.1"
```
Comment (ScoutHarris) [inline Gridicons.podspec]: Shouldn't this be `1.1.0-beta.1`? (According to our new versioning scheme.)

File: Gridicons/Gridicons.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -282,7 +283,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (ScoutHarris) [inline Gridicons/Gridicons.xcodeproj/project.pbxproj]: WP is still at iOS 11.0. Will this conflict?

**Review by ScoutHarris — APPROVED** [review]
✅ See the new P2 icon listed in the demo

I left one comment and one question. But otherwise :shipit: .

**Review by Gio2018 — APPROVED** [review]
Agree with @ScoutHarris comments, other than that :shipit:!

File: Gridicons/Gridicons.xcodeproj/project.pbxproj (hunk 2 above)
Comment (mindgraffiti) [reply Gridicons/Gridicons.xcodeproj/project.pbxproj]: Freudian slip. I'm in denial that WPiOS still supports iOS 11 😛

**Comment by mindgraffiti** [comment]
Updated to standard podspec versioning (1.1.0-beta.1 instead of 1.0.3-beta.1) and changed target OS version to 11.0 instead of 12.0. Will merge after tests pass. Thank you @Gio2018 and @ScoutHarris for the reviews!

**Pull request merged**